src/main/resources/org/springframework/integration/aws/config/spring-integration-aws-1.1.xsd

```diff
@@ -433,6 +433,144 @@
 		</xsd:complexType>
 	</xsd:element>
 
+	<xsd:element name="s3-inbound-streaming-channel-adapter">
+		<xsd:annotation>
+			<xsd:documentation>
+				Configures a 'SourcePollingChannelAdapter' Endpoint for the
+				'org.springframework.integration.aws.inbound.S3InboundStreamingMessageSource'.
+			</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:extension base="base-inbound-adapter-type">
+					<xsd:attribute name="comparator" type="xsd:string">
+						<xsd:annotation>
+							<xsd:documentation><![CDATA[
+								Specify a Comparator to be used when ordering Files. If none is provided, the
+								order in which files are processed is the order they are received from the
+								S3 server. The generic type of the Comparator must be 'S3FileInfo'.
+							]]></xsd:documentation>
+						</xsd:annotation>
+					</xsd:attribute>
+				</xsd:extension>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+
+	<xsd:complexType name="base-inbound-adapter-type">
```

Comment:
I guess we can reuse this new type from the `s3-inbound-channel-adapter`.
Nice!

But, right, it can be addressed in separate commit/PR.